src/lib.rs

```diff
@@ -428,6 +428,68 @@ const KSDATAFORMAT_SUBTYPE_PCM: [u8; 16] = [0x01, 0x00, 0x00, 0x00, 0x00, 0x00,
 const KSDATAFORMAT_SUBTYPE_IEEE_FLOAT: [u8; 16] = [0x03, 0x00, 0x00, 0x00, 0x00, 0x00, 0x10, 0x00,
                                                    0x80, 0x00, 0x00, 0xaa, 0x00, 0x38, 0x9b, 0x71];
 
+
+impl WavSpec {
+    /// Get "stand-alone" wav header representing infinite or unknown size wav file.
+    /// Use this if you need to write audio data to non-seekable sinks (like stdout).
+    ///
+    /// Actual samples are supposed to be written without `hound`'s help.
```

Comment:
We could point out that you can still use `hound::Sample::write()`. Writing samples manually is easy enough, except that 8 bits per sample is an edge case that works differently, and `Sample::write` handles all cases transparently.